Blockzilla/BrowserViewController.swift

```diff
@@ -1089,6 +1090,21 @@ extension BrowserViewController: OverlayViewDelegate {
         self.find(query, function: "find")
     }
     
+    func overlayView(_ overlayView: OverlayView, didAddToAutocomplete query: String) {
+        let autocompleteSource = CustomCompletionSource()
+        
+        switch autocompleteSource.add(suggestion: query) {
+        case .error(.duplicateDomain):
+            break
+        case .error(let error):
+            guard !error.message.isEmpty else { return }
+            Toast(text: error.message).show()
+        case .success:
+            Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.change, object: TelemetryEventObject.customDomain)
+            Toast(text: UIConstants.strings.autocompleteCustomURLAdded).show()
+        }
```

Comment:
After pressing the button, let's dismiss the overlay view (once again mimicking android behavior here).